snippets/assertions.json

```diff
@@ -0,0 +1,72 @@
+{
+	"t.pass()": {
+		"prefix": "t.pass",
+		"body": "t.pass(${1:'${2:message}'});",
+		"description": "Passing assertion."
```

Comment:
I have no clue. Might better remove it then. They aren't present in the `test` snippets either.